BetterPasswordPlugin.inc.php

```diff
@@ -208,17 +236,62 @@ function checkPassword($hookName, $args) {
 			}
 		}
 	}
-
+	/**
+	 * Create a temporary file with the data of password blacklists.
+	 * @return string temporary filename string
+	 */
+         function handleTempFile() {
+            import('lib.pkp.classes.file.PrivateFileManager');
+            $fileMgr = new PrivateFileManager();
+            $siteDao = DAORegistry::getDAO('SiteDAO');
+            $site = $siteDao->getSite();
+            $minLengthPass = $site->getMinPasswordLength();
+            $tempFileDir = realpath($fileMgr->getBasePath()) . DIRECTORY_SEPARATOR . 'passTmp';
+            if (!$fileMgr->fileExists($tempFileDir, 'dir')) {
+			$success = $fileMgr->mkdirtree($tempFileDir);
+			if (!$success) {
+				// Files directory wrong configuration?
+				assert(false);
+				return false;
```

Comment:
What are we going to do with the `false` return value in the parent?  You can drop the `assert(false)` it is obsolete / dev-only PHP style.